src/tools/ceph_kvstore_tool.cc

```diff
@@ -558,8 +559,14 @@ int main(int argc, const char *argv[])
     }
 
   } else if (cmd == "store-crc") {
-    uint32_t crc = st.traverse(string(), true, NULL);
-    std::cout << "store at '" << path << "' crc " << crc << std::endl;
+    if (argc < 4) {
+      usage(argv[0]);
+      return 1;
+    }
+    std::ofstream fs(argv[4]);
+    uint32_t crc = st.traverse(string(), true, &fs);
+    fs.close();
```

Comment:
"fix" is too vague for me. could you expand a little bit? also, `fs` will be closed when it's restructured. i don't think this call is really necessary.

Reply:
thank you, kefu, updated!